x/feedist/codec.proto

```diff
@@ -0,0 +1,57 @@
+syntax = "proto3";
+
+package feedist;
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+
+
+// Revenue represents an account with funds collected from the fees. This is a
+// temporary account used for storing fees that are later distributed between
+// the owners.
+message Revenue {
```

Comment:
These definitions all look quite good to me, and great comments.
I would add an `IssueMsg`, as we want a way to create these outside of the genesis account (as mentioned in pr comments)

Reply:
I have added a [`NewRevenueMsg`](https://github.com/iov-one/weave/pull/316/commits/768852c39888bbf00b520e665cbfbcab8d38c68f#diff-e8a6b07f9af21a9e0791ee2147b9466fR40) message. It also makes the tests much easier to write.